portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,94 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <link rel="stylesheet" href="node_modules/bulma/css/bulma.css">
     <link rel="stylesheet" href="style.css">
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <section class="hero is-light is-large">
+        <div class="hero-head">
+    <nav class="navbar">
+      <div class="container">
+        <div class="navbar-brand">
+          <a class="navbar-item">
+            <h1>Iris Liu</h1>
+          </a>
+          <span class="navbar-burger burger" data-target="navbarMenuHeroB">
```

Comment:
I assume the 3 spans within are the lines of the hamburger button? Maybe consider a comment here.

Also, what does the data-target do? You haven't included any JS in this page except for Font Awesome so I'm not sure what it's for.

Reply:
Re-opening this comment, I think this is still unresolved.

I also don't see the hamburger button when I build, am I missing something/building it incorrectly? This is what I see:
![image](https://user-images.githubusercontent.com/20651047/83101411-23bebd80-a067-11ea-9aeb-2e1e3c79101c.png)

Reply:
![image](https://user-images.githubusercontent.com/42590515/83162740-286f8a00-a0d8-11ea-8ae6-497146f551ae.png)

Reply:
so i think this is not working on your device maybe? I have deleted this button and spans came with it